dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -363,7 +372,7 @@ public void ContentFallbackFalse_PageFallbackTrue_PageEnglish_ViewEnglishContent
                     .setPageMode(PageMode.LIVE)
                     .build(),
                 mockRequest, mockResponse);
-        Assert.assertTrue("ENG = "+html , html.contains("content2content1"));
+        Assert.assertTrue("ENG = "+html , html.contains("content1") && html.contains("content2"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "ENG = " appears 6 times in this file; the first occurrence is on line 375](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5316867)